tsdb/chunkenc/histo_meta.go

```diff
@@ -32,29 +42,36 @@ func putHistoChunkMetaSpans(b *bstream, spans []histogram.Span) {
 	}
 }
 
-func readHistoChunkMeta(b *bstreamReader) (int32, float64, []histogram.Span, []histogram.Span, error) {
+func readHistoChunkMeta(b *bstreamReader) (bool, int32, float64, []histogram.Span, []histogram.Span, error) {
```

Comment:
With the bool and the many return values in general, perhaps named returns would make sense here, purely for documentation purposes.

And maybe, very maybe even use naked returns to make the many `return false, 0, 0, nil, nil, err` shorter. But that's a trade-off with the issues of naked returns…